Include/mydatatypes.h

```diff
@@ -0,0 +1,62 @@
+
+//  UC Berkeley - MSSE Program
+//  Chem 279-B  Introduction to Software Engineering
+//  Fall 2022
+//
+// his file contains the custom datatypes used in the Cellular Automata project
+//
+#pragma once    // Ensures that this file is only included once
+                // during compilation
+
+#include <array>
+using namespace std;
+
+enum neighborhood{VonNeumann, Moore};
+
+enum boundary{None, Periodic, Walled, CutOff};
+
+enum rule{Majority, Parity};
+
+class CellularAutomata         // The CA datastructure
+{
+    public:   
+
+        boundary b_type;     //enum code to hold boundary
```

Comment:
b_type is a bit unclear. Maybe don't shorted it and just write boundary_type

Reply:
I will change this, so don't bother merging the code yet. I'll submit another request.